astropy/time/core.py

```diff
@@ -1659,9 +1659,9 @@ def __array_function__(self, function, types, args, kwargs):
     def ptp(self, axis=None, out=None, keepdims=False):
         """Peak to peak (maximum - minimum) along a given axis.
 
-        This is similar to :meth:`~numpy.ndarray.ptp`, but adapted to ensure
-        that the full precision given by the two doubles ``jd1`` and ``jd2``
-        is used.
+        This method is similar to the :func:`numpy.ptp` function, but
```

Comment:
good catch, though I'm thinking the phrasing might become increasingly confusing as numpy 1.x fades away in usage. How about this ?
```suggestion
        This method is similar to the :func:`numpy.ptp` legacy function, but
```

Reply:
The `np.ndarray.ptp()` method has been removed, but the `np.ptp()` function is still there and I am not aware of any plans to remove it, so it's not a legacy function.

Reply:
see ? it *is* confusing 😅 
(thanks for clarifying)

Reply:
We should not mention deprecations in docstrings because our `deprecated` decorator edits docstrings automatically. If the default text `deprecated` adds to the docstring is not good enough then a custom text should be specified through the decorator's `message` argument.

This pull request was meant to be a quick fix to prevent documentation builds from failing, so discussing how to phrase the deprecation message delays merging and causes problems in other pull requests.